src/sessions/yt-music/YTMusicVolumeControl.ts

```diff
@@ -29,27 +28,15 @@ export class YTMusicVolumeControl implements VolumeControl {
    */
   public async setVolume(volume: number) {
     if (volume < 0 || volume > 1) return;
-    await this.page.$eval(
-      'video',
-      (el: HTMLVideoElement, volume) => {
-        el.volume = volume;
+    await this.page.evaluate(
+      (volume: number) => {
+        const volumeSlider: HTMLElement = document.querySelector('#volume-slider');
+        // @ts-ignore becuase YT music uses a custom HTML element for this.
```

Comment:
pretty sure they just extended an HTMLInputElement - maybe specifying it as that type will work? If not you can always do something like HTMLElement & { value: number }... Not as clean